packages/lib-classifier/src/plugins/drawingTools/experimental/models/marks/FreehandLine/FreehandLine.js

```diff
@@ -210,7 +240,7 @@ const FreehandLineModel = types
     trim(startIndex, endIndex) {
       const clippedPoints = [ ...self.points.slice(endIndex), ...self.points.slice(0, startIndex + 1)]
       self.clipPath = clippedPoints.map(({ x, y }) => ({ x, y }))
-      self.points.splice(0, startIndex + 1)
+      self.points.splice(0, startIndex)
```

Comment:
The second argument to [`slice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) is an array index, but the second argument to [`splice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) is the number of points that you want to remove from the array. Here, we remove `startIndex` points from the beginning of the array, which moves the first point to `startIndex`.